website/docs/d/batch_pool.html.markdown

```diff
@@ -324,6 +344,8 @@ A `network_security_group_rules` block exports the following:
 
 * `source_address_prefix` - The source address prefix or tag to match for the rule.
 
+* `source_port_ranges` - The source port ranges to match for the rule.
```

Comment:
we should give an example here?

Reply:
Example provided in resource https://github.com/hashicorp/terraform-provider-azurerm/pull/18436/files#diff-d2ab6ef0f53d03261fa2ae2592028e23f85e2838c8037f73bd939afa38573d59R483